examples/minimal-expo/metro.config.js

```diff
@@ -0,0 +1,47 @@
+const { getDefaultConfig } = require("expo/metro-config");
+const { FileStore } = require("metro-cache");
+const path = require("path");
+
+module.exports = withTurborepoManagedCache(
```

Comment:
Ensure you add `config.resolver.unstable_enablePackageExports = true;` to enable package exports. They're a bit buggy in Metro, but @robhogan has been looking into them a bit recently so it may improve.